### PR TITLE
added subgraph studio endpoints

### PR DESCRIPTION
**Pull request by calvinkei**
tested and it looks fine! fetching data correctly

### CODE REVIEW

**Comment by dimsome** [comment]
Looks like it works fine

File: libs/base/src/lib/context/NetworkProvider.tsx [hunk 1]
```diff
@@ -170,20 +170,23 @@ const ETH_MAINNET: EthereumMainnet = {
...
     questbook: ['https://europe-west1-mstable-questbook.cloudfunctions.net/questbook'],
-    merkleDrop: [graphHostedEndpoint('mstable', 'mstable-merkle-drop')],
+    merkleDrop: [
+      'https://api.studio.thegraph.com/query/32034/mstable-ecosystem/v0.1.0',
+      graphHostedEndpoint('mstable', 'mstable-merkle-drop'),
+    ],
     snapshot: ['https://hub.snapshot.org/graphql'],
     feeders: [
       // Temporary preview URL because indexers haven't picked up the new version...
       'https://api.studio.thegraph.com/query/32034/mstable-feeder-pools-and-vaults/v0.0.13',
```
Comment (rafaelugolini) [inline libs/base/src/lib/context/NetworkProvider.tsx:185]: @calvinkei did you also deploy this one?

Comment (calvinkei) [reply libs/base/src/lib/context/NetworkProvider.tsx:185]: no since I saw this on studio already

Comment (rafaelugolini) [reply libs/base/src/lib/context/NetworkProvider.tsx:185]: @calvinkei I would recommend to publish a new version and test with it

**Comment by calvinkei** [comment]
close https://github.com/mstable/mStable-apps/issues/205

File: libs/base/src/lib/context/NetworkProvider.tsx [hunk 2]
```diff
@@ -170,20 +170,28 @@ const ETH_MAINNET: EthereumMainnet = {
   gasStationEndpoint: 'https://gas.mycryptoapi.com/',
   gqlEndpoints: {
     protocol: [
-      graphHostedEndpoint('mstable', 'mstable-protocol'),
-      graphMainnetEndpoint('0x26cf67040678eb0f5654c9cbaad78dc1694cbafa', 0, process.env.NX_PROTOCOL_SUBGRAPH_API_KEY as string),
+      'https://api.studio.thegraph.com/query/32034/mstable-protocol-2/v1.0.1',
```
Comment (toniocodo) [inline libs/base/src/lib/context/NetworkProvider.tsx:173]: do we need to keep this endpoint?

File: libs/base/src/lib/context/NetworkProvider.tsx [hunk 3]
```diff
@@ -148,8 +148,8 @@ const etherscanUrl =
     }
   }
 
-const graphMainnetEndpoint = (subgraphId: string, version: number, apiKey: string): string =>
-  `https://gateway.thegraph.com/api/${apiKey}/subgraphs/id/${subgraphId}-${version}`
+const graphMainnetEndpoint = (subgraphId: string, apiKey: string): string =>
+  `https://gateway.thegraph.com/api/${apiKey}/subgraphs/id/${subgraphId}`
```
Comment (toniocodo) [inline libs/base/src/lib/context/NetworkProvider.tsx:152]: wouldn't it be better to directly call `process.env.NX_SUBGRAPH_API_KEY` and remove arg?

Comment (calvinkei) [reply libs/base/src/lib/context/NetworkProvider.tsx:152]: we might use different api keys for different subgraphs, so i think keeping the arg here for flexibility makes sense

File: libs/base/src/lib/context/NetworkProvider.tsx (hunk 2 above)
Comment (calvinkei) [reply libs/base/src/lib/context/NetworkProvider.tsx:173]: this is added for now to avoid high cost on calling published subgraphs

**Review by toniocodo — APPROVED** [review]

**Review by dimsome — APPROVED** [review]

**Pull request merged**